README.md

```diff
@@ -2,7 +2,7 @@
 
 ![ethernaut](./public/imgs/bann.jpeg)
 
-<p>The ethernaut is a Web3/Solidity based wargame inspired in <a href="https://overthewire.org" target="_blank" rel="noopener noreferred">overthewire.org</a> and in the comic <a href="https://en.wikipedia.org/wiki/The_Eternaut" target="_blank" rel="noopener noreferred">El Eternauta</a>, to be played in the Ethereum Virtual Machine. Each level is a smart contract that needs to be 'hacked'.</p>
```

Comment:
Taking inspiration on an existing cultural work is perfectly legal, so this doesn't have to be removed. I would leave it.